src/App.tsx

```diff
@@ -1,60 +1,151 @@
 import './App.scss';
+import { useEffect, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [titleInput, setTitleInput] = useState('');
+  const [userSelect, setUserSelect] = useState('');
+  const [checkInput, setCheckInput] = useState(false);
+  const [showSpanTitle, setShowSpanTile] = useState(false);
+  const [showSpanUser, setShowSpanUser] = useState(false);
+  const [visibleTodos, setVisibleTodos] = useState([...todosFromServer]);
+
+  useEffect(() => {
+    if (titleInput.length !== 0 && +userSelect !== 0) {
+      setCheckInput(true);
+    }
+
+    if (titleInput.length !== 0) {
+      setShowSpanTile(false);
+    }
+
+    if (+userSelect !== 0) {
+      setShowSpanUser(false);
+    } else {
+      setCheckInput(false);
+    }
+  });
```

Comment:
It seems that useEffect is redundant here. Move it to a function or add a dependency array

Reply:
Not fixed

Reply:
@loralevitska @Anastasiia-Svintsova I don't understand what you mean
Can you please explain with more details